providers/curation/github.js

```diff
@@ -320,6 +328,25 @@ class GitHubCurationService {
     }
   }
 
+  async getPrComponents(number) {
```

Comment:
Nice. Some naming comments (naming is hard)

* getPrComponents => getChangedDefinitions
* result => files
* prComponents => changedCoordinates
* fileObj => remove and just files[i] in the one place
* prVersion => prDefinitions
* masterVersion => currentDefinitions
* relevantRevisions => changedRevisions